docs/monoDrive_home/Vehicle-Configuration.md

```diff
@@ -55,14 +55,30 @@ Carpaint_Red
 Carpaint_Silver
 Carpaint_White
 ```
-When selecting a color programmatically (i.e. using any client) the user can assign the color to use for the ego vehicle using the `vehicle.json` with the `body` tag. 
+
+### Closed Loop Mode
+
+When selecting a color programmatically (i.e. using any client) in Closed loop mode, the user can assign a color for the ego vehicle using the `simulator.json` in the `body` tag. 
```

Comment:
The replay mode uses the `simulator.json`  file and the closed loop mode use the scenario file